Backend/dev/migrations/dynamic-offer-driver-app/0232-adding-disability-tag-to-search-request-table.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE atlas_driver_offer_bpp.search_request ADD COLUMN disability_tag text;
```

Comment:
This query is present in another file, you can remove it from here and combine all these in one file